pkgs/by-name/je/jellyfin-tui-rs/package.nix

```diff
@@ -0,0 +1,51 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  rustPlatform,
+  pkg-config,
+  mpv-unwrapped,
+  sqlite,
+  versionCheckHook,
+  mpris ? stdenv.isLinux,
+}:
+
+rustPlatform.buildRustPackage {
+  pname = "jellyfin-tui-rs";
+  version = "0.1.2";
+  src = fetchFromGitHub {
+    owner = "owo-uwu-nyaa";
+    repo = "jellyfin-tui-rs";
+    rev = "594fb385bbf2e245ce8f3650713219fa43772be2";
```

Comment:
I suggest using `tag` for releases, and `rev` only for unstable commits (when not tagged yet but required).